plugins/Bitcrush/Bitcrush.cpp

```diff
@@ -153,7 +153,7 @@ bool BitcrushEffect::processAudioBuffer( SampleFrame* buf, const fpp_t frames )
 	// read input buffer and write it to oversampled buffer
 	if( m_rateEnabled ) // rate crushing enabled so do that
 	{
-		for( int f = 0; f < frames; ++f )
+		for (auto f = std::size_t{0}; f < frames; ++f)
```

Comment:
Wouldn't it be better to do
```suggestion
		for (size_t f = 0; f < frames; ++f)
```
All this extra syntax additions seems to confuse people for little benefit.

Reply:
Additions to the language are going to be used as we adopt newer C++ standards. I don't think we should restrict ourselves to older C++ versions on the basis that people may not have learnt the newer ones yet. If you come across something you're not familiar with, look it up on the reference wiki, or in the standard, or on Stack Overflow, or ask an LLM, or ask us on Discord, etc. Given that your suggestion is correct, it looks like you've figured this one out already, however. I'm not making other people write their code like this, but it's how I intend to do so.

When we can use C++23, `std::size_t{0}` can be replaced with `0uz`. For now, it can be thought of as a rather verbose literal.

Reply:
I was complaining about the verbosity. But yeah, since you don't force others, it's fine.